package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "thrift-pool",
```

Comment:
right, forgot about the other `thrift-pool`... `thrift-connection-pool`, `thrift-client-wrapper`